webapp/src/main/new/dataService.ts

```diff
@@ -21,10 +41,18 @@ function postData(
   });
 }
 
-export function insertStems(stems: Stem[]): Promise<void> {
+export function getCensus(): Promise<Stem[]> {
+  // TODO: Remove the access code and generate a new one
+  const url =
+    "https://forestgeodataapi.azurewebsites.net/api/Census?code=0kOITiOaxOVEYuCNzIc9V6uwhTz41qPvf92RgFF6bBfHQAZcsavyPA==";
+
+  return getData(url);
+}
+
+export function insertCensus(stems: Stem[]): Promise<void> {
   // TODO: Remove the access code and generate a new one
   const url =
-    "https://forestgeo-middletier.azurewebsites.net/api/InsertItems?code=D2BT0IK717/vf8YFkdNe2bXTzYMTZkKty4qlTk0FdpxCBJAF146xzw==";
+    "https://forestgeodataapi.azurewebsites.net/api/Census?code=xqRLaAgGAAQkMuMUwFu//JKC7BCEraubIlmfWOZsSBs4IsdbPDMF8w==";
```

Comment:
This looks like a secret... We can't check this into our repo. We'll need to come up with some authentication system in the future, but for now can we just change this to `"<----- insert connection string here ----->"`